### PR TITLE
Organize imports

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

File: appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/deployment/annotation/handlers/ConfigPropertyHandler.java [hunk 1]
```diff
@@ -508,7 +507,7 @@ public static void processParent(Class claz, Set configProperties) {
             if (property != null) {
                 String result = validateMethod(m, property);
                 if (!result.equals(SUCCESS)) {
-                    throw new IllegalStateException(result);
+                    throw new java.lang.IllegalStateException(result);
```
Comment (dmatej) [inline appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/deployment/annotation/handlers/ConfigPropertyHandler.java:510]: Why???

File: qa/src/main/resources/org/glassfish/qa/config/checkstyle/checkstyle.xml [hunk 2]
```diff
@@ -82,6 +82,7 @@
         <!-- Checks for imports                              -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="IllegalImport" /> <!-- defaults to sun.* packages -->
+        <module name="RedundantImport" />
```
Comment (dmatej) [inline qa/src/main/resources/org/glassfish/qa/config/checkstyle/checkstyle.xml:85]: I agree, if it is fixed everywhere, but not by using package names like java.lang.Object in code. If there are conflicts, ie. org.glassfish.Object, I would prefer JDK in imports and conflicting class with full name in code (except if there would be 100:1 occurences).

File: appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/deployment/annotation/handlers/ConfigPropertyHandler.java (hunk 1 above)
Comment (pzygielo) [reply appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/deployment/annotation/handlers/ConfigPropertyHandler.java]: Reverted to not-FQ name.

File: qa/src/main/resources/org/glassfish/qa/config/checkstyle/checkstyle.xml (hunk 2 above)
Comment (pzygielo) [reply qa/src/main/resources/org/glassfish/qa/config/checkstyle/checkstyle.xml:85]: Conflicts resolved by replacing on-demand imports with individual ones.

**Comment by pzygielo** [comment]
reworked

**Review by dmatej — APPROVED** [review]

**Pull request merged**